github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-postsubmits.yaml

```diff
@@ -0,0 +1,53 @@
+postsubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: main-kubemacpool
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"
+      branches:
+        - main
+      cluster: prow-workloads
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20201119-a5880e0
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - "cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io && make container docker-push"
+            # docker-in-docker needs privileged mode
+            securityContext:
+              privileged: true
+    - name: release-kubemacpool
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"
+      branches:
+        - release-v\d+\.\d+
```

Comment:
This is what we do at knmstate to do the release https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-postsubmits.yaml#L59

Reply:
yeah that's what I initially did, 
but, since knmstate branches syntax are a bit different than kmp's (release-v0.20 vs release-0.20), I decided to copy from cdi which uses the same branch syntax as KMP...

Reply:
but we are looking for a tag not a branch, prow is parsing tags too there we need to match the tag we only release when a tag is pushed.